scripts/clientValidation.js

```diff
@@ -138,10 +119,10 @@ function lessThan255(input, valClass) {
 function lessThan14(input, valClass) {
     $('.err').remove();
     let isValid = false;
-    if (input.val().trim() != "") {
+    if (input.val().trim() != "" && input.val().length !=14) {
```

Comment:
Should this be input.val().length < 14? Since it's checking to see if it's less than 14, not just not equal to 14?